File: Exceções - IF Quest/Java/Item.java
public class Item {
    private String nome;
    private int bonus;

    public Item(String nome, int bonus) {
        this.nome = nome;
        setBonusItem(bonus);
    }

    // Getters
    public String getNomeItem() {
        return nome;
    }

    public int getBonusItem() {
        return bonus;
    }

    // Setters
    public void setNomeItem(String nome) {
        this.nome = nome;
    }

    public void setBonusItem(int bonus) {
        if (bonus >= 0) {
            this.bonus = bonus;
        } else {
            throw new IllegalArgumentException(
                "Bônus inválido! Insira um valor maior ou igual a 0 (valor informado: " + bonus + ")"
            );
        }
    }

    // Metodos
    public void descricao() {
        System.out.printf("%s (+%d)%n", nome, bonus);
    }

}

File: Interface Gráfica - Tela de Criação de Herói/Python/item.py
class Item:
    def __init__(self, nome, bonus):
        self.__nome = nome
        self.setBonusItem(bonus)

    # Getters
    def getNomeItem(self):
        return self.__nome

    def getBonusItem(self):
        return self.__bonus

    # Setters
    def setNomeItem(self, nome):
        self.__nome = nome

    def setBonusItem(self, bonus):
        if bonus >= 0:
            self.__bonus = bonus
        else:
            raise ValueError(
                f"Bônus inválido! Insira um valor maior ou igual a 0 (valor informado: {bonus})"
            )

    # Metodos
    def descricao(self):
        print(f"{self.__nome} (+{self.__bonus})")
